internal/exportimport/importer.go

```diff
@@ -137,10 +146,15 @@ func (s *Server) runImport(ctx context.Context, config *model.ExportImport) erro
 		if err := s.exportImportDB.CompleteImportFile(ctx, file, status); err != nil {
 			logger.Errorf("failed to mark file completed", "file", file, "error", err)
 		}
+
+		// TODO: Do we want to keep trying files rather than returning?
+		//       This seems like we quit awfully early.
+		if err != nil {
+			return fmt.Errorf("error processing import file: %w", err)
```

Comment:
I think we want to keep processing.

There is a race condition where the oldest files might be cleaned up before the index is rewritten and the oldest files may be attempted to be imported first.

Reply:
fixed.